spi.java.hints/src/org/netbeans/spi/java/hints/JavaFixUtilities.java

```diff
@@ -1337,6 +1336,7 @@ private void rewrite(Tree from, Tree to) {
 
         OPERATOR_PRIORITIES.put(Kind.ARRAY_ACCESS, 1);
         OPERATOR_PRIORITIES.put(Kind.METHOD_INVOCATION, 1);
+        OPERATOR_PRIORITIES.put(Kind.MEMBER_REFERENCE, 1);
```

Comment:
Hello @lbruun,

Thank you for your review. You are right that this addition affects requiresParenthesis(). It actually fixes an issue with requiresParenthesis().

I have added a new unit test which breaks without this extra line. Instead of rewriting `Test::identity` to `Test::toString`, with the line adding `Kind.MEMBER_REFERENCE` to the `OPERATOR_PRIORITIES` map, it was rewritten to `(Test)::toString` without.